### PR TITLE
refactor: restructure x/consortium

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

closes: #489 

## Motivation and context

Refer to #489.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes. [no need]
- [ ] I have updated the documentation accordingly. [no need]
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml` [no need]

### CODE REVIEW

**Comment by 0Tech** [comment]
I should add tests to increase the coverage. Let me handle it on the separate issue later.

File: x/consortium/keeper/keys.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	validatorAuthKeyPrefix = []byte{0x01}
+)
+
+// validatorAuthKey key for a specific validator from the store
+func validatorAuthKey(valAddr sdk.ValAddress) []byte {
+	key := make([]byte, len(validatorAuthKeyPrefix)+len(valAddr))
+	copy(key, validatorAuthKeyPrefix)
+	copy(key[len(validatorAuthKeyPrefix):], valAddr)
+	return key
+}
```
Comment (zemyblue) [inline x/consortium/keeper/keys.go:24]: This function doesn't check if this key is multisig or not. 
Do you add this validating function later?

File: x/consortium/keeper/genesis.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		if err := k.SetValidatorAuth(ctx, auth); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (k Keeper) ExportGenesis(ctx sdk.Context) *consortium.GenesisState {
+	return &consortium.GenesisState{
```
[the remark this answers is not in the document]
Comment (0Tech) [reply x/consortium/keeper/genesis.go:39]: There would be two options, removing the `NewGenesisState` or using the function here. I will go with the former.

File: x/consortium/genesis.go [hunk 3]
```diff
@@ -2,42 +2,38 @@ package consortium
...
-	"github.com/line/lbm-sdk/x/consortium/types"
-
-	stakingtypes "github.com/line/lbm-sdk/x/staking/types"
+	sdkerrors "github.com/line/lbm-sdk/types/errors"
 )
 
-func InitGenesis(ctx sdk.Context, keeper keeper.Keeper, sk types.StakingKeeper, data *types.GenesisState) error {
-	keeper.SetParams(ctx, data.Params)
+// NewGenesisState creates a new GenesisState object
+func NewGenesisState(params *Params, validatorAuths []*ValidatorAuth) *GenesisState {
```
[the remark this answers is not in the document]
Comment (0Tech) [reply x/consortium/genesis.go:9]: I will remove the function.

File: x/consortium/keeper/keys.go (hunk 1 above)
Comment (0Tech) [reply x/consortium/keeper/keys.go:24]: It has nothing to do with the new feature. 
The function would be implemented on another issue.

**Review by zemyblue — APPROVED** [review]
LGTM

**Review by dudong2 — APPROVED** [review]

**Pull request merged**